### ISSUE TITLE
OCC Access Conditions Not Specified in NIST Compliant Profile

### ISSUE THREAD

**Issue by dmercer-google** [makinako/OpenFIPS201#67]
The [NIST compliant Profile for key objects](https://openfips201.atlassian.net/wiki/spaces/OD/pages/491612/Appendix+-+NIST+Compliant+Profile#6-Key-Object-Scripts) does not specify any OCC conditions.  While I realize that OCC is not currently supported It would be good to address them in the provisioning scripts so that when it is people wont have to change their scripts.  Also note my prior two bugs about ```occ``` vs ```occAlways``` and ```VCI```.  This should be added to the profile when addressed, even though they are not yet implemented.

**Comment by makinako** [comment]
Agreed, once we have closed issue #65 I will make the changes to the default script to include the correct occ settings.

**Comment by makinako** [comment]
Closing as references to OCC have been removed in the FIPS version until we build it in.
